components/ChipTextField/examples/MDCChipTextField.h

```diff
@@ -45,4 +84,12 @@
  */
 - (void)appendChipWithText:(nonnull NSString *)text NS_SWIFT_NAME(appendChip(text:));
 
+/**
+ Set a chip view at particular index to be selected or not.
+
+ @param selected selection status to set
+ @param index The index of the chip view to set the selection status
+ */
+- (void)setChipViewSelected:(BOOL)selected atIndex:(NSInteger)index;
```

Comment:
What is the reason for this API?  Can we add an explanation to the design doc?

Reply:
Removed this API in 0003b1507a139f7d11899e583f895bac70dd94e6